Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -6,10 +6,13 @@ import Foundation
 import UIKit
 import Shared
 
-class IntroViewController: UIViewController {
+class IntroViewController: UIViewController, OnboardingViewControllerProtocol {
+
     private var viewModel: IntroViewModel
     private let profile: Profile
     private var onboardingCards = [OnboardingCardViewController]()
+    var notificationCenter: NotificationProtocol = NotificationCenter.default
+    var didFinishClosure: (() -> Void)?
```

Comment:
Wondering if this should be renamed as it's not clear what finished without looking at the code 🤔